File: ws1920.php
<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8">
    <title> ws1920 php Motta </title>
</head>
<body>
<?php
$name = $_SERVER['QUERY_STRING'];
$searchTerm = "";
?>
<form method='post' action="ws1920.php?<?php echo $name ?>">
    <label>Hello <?php echo $name ?> what are you looking for</label>
    <label><Input type="text" name="text"  ></label>
    <label><Input type="submit" name="submit"></label><br>

    <?php

        if ($_SERVER['REQUEST_METHOD'] == 'POST') {


            if($_POST['text'] !== "")
            {
                $searchTerm=$_POST['text'];
                echo '<a href="https://www.google.com/search?q='.$searchTerm.'">Link</a>';

            } else  echo "no search term";
        }

        ?>

</form>
</body>
</html>

File: ws1920.c
#include <stdio.h>
#include <stdlib.h>
int main ()
{
printf("Content-Type:text/html\n\n");
printf("<!DOCTYPE html>\n");
printf("<html><head>\n");
printf("<meta charset='UTF-8'><title>ws1920.c Motta</title>\n");


printf("<body><a href='http://info.mi.hs-offenburg.de/~ida/cgi-bin/darima/ws1920.c?Tick%s'>Tick%s</a>", getenv("QUERY_STRING"), getenv("QUERY_STRING"));


printf("</body></html>\n");
return 0;}


